sparse_ho/criterion/multiclass_logreg.py

```diff
@@ -15,12 +15,15 @@ class LogisticMulticlass():
         indices of the training set
     idx_val: np.array
         indices of the validation set
-    algo: TODO
-    idx_test: TODO
+    algo: instance of ``sparse_ho.base.AlgoModel``
+        A model that follows the sparse_ho API.
+    idx_test: np.array
+        indices of the test set
 
     Attributes
     ----------
-        TODO
+    dict_models: dictionnary
```

Comment:
```suggestion
    dict_models: dict
```
the python type is called dict